JongHo/app.js

```diff
@@ -42,6 +41,119 @@ app.post("/userCreate", async (request, response) => {
   response.status(201).json({ message: "userCreated" });
 });
 
+//create a post
+app.post("/postCreate", async (request, response) => {
+  const { title, content, user_id, image_url } = request.body;
```

Comment:
```suggestion
  const { title, content, userId, imageUrl } = request.body;
```
- 테이블 내 컬럼명을 작성할 때에는 `언더바(_)`를 활용하여 `snake case`로 작성해 주세요.
- 또한 모든 네이밍 컨벤션이 동일해야 합니다. (CamelCase, snake_case를 섞어서 사용하면 안됨)